File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// Covid19Data SDK exists test

require_once __DIR__ . '/../covid19data_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = Covid19DataSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# Covid19Data SDK exists test

require "minitest/autorun"
require_relative "../Covid19Data_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = Covid19DataSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
